src/Tizen.NUI/src/public/BaseComponents/TextEditor.cs

```diff
@@ -1419,7 +1419,7 @@ public bool Ellipsis
         /// The ellipsis position type when the text size over the layout size.<br />
         /// The ellipsis position: End, Start or Middle.<br />
```

Comment:
The same description is used in other places
```suggestion
        /// Specifies which portion of the text should be replaced with an ellipsis when the text size exceeds the layout size.<br />
        /// The ellipsis position: End, Start or Middle.
```
The last sentence could also be removed, for a person reading the docs it's only 1 click to see details of the type.

Reply:
Done